docs/gitbook/usage/deployment-strategies.md

```diff
@@ -148,10 +148,10 @@ In order to enable non-linear promotion a new parameter was introduced:
 Example:
 
 ```yaml
-canary:
+# canary.yaml
+spec:
   analysis:
-    promotion:
-      stepWeights: [1, 2, 10, 80]
+    stepWeights: [1, 2, 10, 80]
```

Comment:
This should be `stepWeightPromotion`

Reply:
Thanks @stefanprodan , `stepWeightPromotion`  works  when canary yaml apply.
But it doesn't work when I use `maxWeightPromotion` or `stepWeightsPromotion`.

Reply:
Yes so there is single field which can control the promotion: `stepWeightPromotion`. We need to update the docs to use only this field.